### ISSUE TITLE
correct tls module names

### ISSUE THREAD

**Issue by kayiwa** [pulibrary/princeton_ansible#1228]

**Comment by kayiwa** [comment]
Datadog agents reports this.
```
Loading Errors
  ==============
    libwww_tls_check
    ----------------
      Core Check Loader:
        Check libwww_tls_check not found in Catalog

      JMX Check Loader:
        check is not a jmx check, or unable to determine if it's so

      Python Check Loader:
        unable to import module 'libwww_tls_check': No module named libwww_tls_check
```

**Comment by acozine** [comment]
Is this error still happening? How would we reproduce it?

**Comment by acozine** [comment]
Looks like this was fixed (and should have been closed) by #1229.
